packages/flutter_tools/templates/ffi_plugin/android.tmpl/build.gradle.tmpl

```diff
@@ -0,0 +1,41 @@
+group '{{androidIdentifier}}'
```

Comment:
we should add detailed comments to this file to explain what is special about it, how it works, how to extend it, what the pitfalls are, etc.